xbmc/PlayListPlayer.h

```diff
@@ -103,6 +103,11 @@ class CPlayListPlayer : public IMsgTargetCallback
    */
   int GetCurrentPlaylist() const;
 
+  /*! \brief Get currently active playlist or a default for current player
```

Comment:
"Get currently active playlist or return the provided default if none is active"

Does it sounds better?